### PR TITLE
Time backgrounds

### PR DESCRIPTION
**Pull request by goodecf**
Add background images that change based on the user's local time.

### CODE REVIEW

**Comment by goodecf** [comment]
https://5ee80b9ab30fbd00089a48e9--weatherapp18.netlify.app/

File: src/components/Library.js [hunk 1]
```diff
@@ -71,29 +78,50 @@ function Library() {
     getWeather(weather);
   };
 
+  const bgTimeColor = () => {
+    const time = new Date().getHours();
+    // const time = 22;
```
[the remark this answers is not in the document]
Comment (goodecf) [reply src/components/Library.js:83]: the comment?

File: src/components/Library.js [hunk 2]
```diff
@@ -71,29 +78,50 @@ function Library() {
     getWeather(weather);
   };
 
+  const bgTimeColor = () => {
```
[the remark this answers is not in the document]
Comment (goodecf) [reply src/components/Library.js:81]: It's based on where the user is located although where they're searching wouldn't be a bad idea and might make more sense. Guess I was assuming people would be searching their own locations so I just used `new Date().getHours()` to get their time.

File: src/components/Library.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (dozoyo) [reply src/components/Library.js:83]: yes

File: src/components/Library.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (dozoyo) [reply src/components/Library.js:81]: Yeah. The confusion here is figuring out the functionality. 
I forgot that the location access currently doesn't work on firefox, too. Did we file an issue for that?

I think it'd be smart to have it update depending on the search result, and have a default background until someone searches. What do you think?

[the remark this answers is not in the document]
Comment (goodecf) [reply src/components/Library.js:81]: The default background can be what it is now - based on the user's current location when the page is loaded. Then update it when a search is made.

[the remark this answers is not in the document]
Comment (dozoyo) [reply src/components/Library.js:81]: Sounds good to me! I think you'll need to add some code to update it when it searches then. You can handle that in this PR or a new one, whichever you prefer.

**Comment by goodecf** [comment]
@dozoyo https://5eec25a946437d00079abc10--weatherapp18.netlify.app/

**Pull request merged**